publish/aliPublish-updatable-rpms.conf

```diff
@@ -0,0 +1,77 @@
+# vim: set filetype=yaml:
+---
+base_url: https://ali-ci.cern.ch/TARS
+http_ssl_verify: False
+conn_timeout_s: 6.05
+conn_retries: 6
+conn_dethrottle_s: 0.5
+
+architectures:
+  slc7_x86-64:
+    RPM: el7.x86_64
+    include:
+      Control:
+        - ^v[0-9]+\.[0-9]+\.[0-9]+-[0-9]+$
+      flpproto:
+        - ^v[0-9]+\.[0-9]+\.[0-9]+-[0-9]+$
+
+# RPM-specific configuration
+rpm_repo_dir: /repo/UpdRPMS
+
+# What packages to publish
+auto_include_deps: True
+filter_order: include,exclude
+
+# Generate updatable RPMs
+rpm_updatable: True
+
+# Packages older than 7 days will be excluded (limits too many packages published by mistake)
+exclude_older_d: 7
+
+notification_email:
+  server: cernmx.cern.ch
+  package_format: "  %(package)s %(version)s\n"
+  success:
+    body: |
+      Dear ALICE fellows,
+
+        Updatable RPM %(package)s %(version)s for architecture %(arch)s was created.
+
+      To install (you might need to force-refresh your cache):
+
+        yum -v clean expire-cache
+        yum install -y alisw-%(package)s
+
+      To use the newly created package:
+
+        aliswmod enter %(package)s/%(version)s
+
+      You can find the full set of instructions (including repo configuration) here:
+
+        https://aliceo2group.github.io/binaries
```

Comment:
this is not the correct URL any more. It should read "https://aliceo2group.github.io/quickstart/binaries.html"

Reply:
Thanks for the heads-up; if this works, we should also update the documentation with instructions for all the three RPM repositories.